src/services

```diff
@@ -587,9 +587,9 @@ p({
   "name":"addressParser",
   "lazy":false,
   "serve":false,
-  "service":{
-    "class":"foam.nanos.geocode.GoogleMapsAddressParser",
-  }
+  "serviceScript": """
+    new foam.nanos.geocode.GoogleMapsAddressParser.Builder().build();
```

Comment:
You're going to have to pass "x" into the Builder method.

Reply:
And return?? I think

Reply:
> And return?? I think

I thought so too, but looked it up and found this:

> Methods with unspecified return types may return any type of object (as in the previous example). Alternatively they may also simply issue a "return;" without a value, in which case the effective type of the method is "void" (no type). In either case, the return statement is optional. If the method does not perform an explicit "return" statement and the return type is not explicitly set to void, the value of the last statement or expression in the method body becomes the return value (and must adhere to any declared return typing). - http://www.beanshell.org/manual/methods.html

So BeanShell is funky and doesn't require an explicit return from methods. Although I don't know if a script counts as a method, so we should probably confirm that it works.